### PR TITLE
mraa: update to version 0.10.1

### PR DESCRIPTION
**Pull request by anujm1**
* Fix packaging errors related to incorrect paths for
  python site package and lib dirs when python and node
  bindings are enabled.
* SKIP_RPATH to make sure mraa.node isn't installed with
  RPATH pointing to local dirs.
* Fix issues with mosquitto

Signed-off-by: Anuj Mittal <anujx.mittal@intel.com>

### CODE REVIEW

File: recipes-connectivity/mosquitto/mosquitto_1.4.bb [hunk 1]
```diff
@@ -23,7 +25,7 @@ do_compile() {
 
 do_install() {
     oe_runmake install DESTDIR=${D}
-    install -d ${D}${libbir}
+    install -d ${D}${libdir}
```
Comment (tripzero) [inline recipes-connectivity/mosquitto/mosquitto_1.4.bb]: yikes!

**Comment by tripzero** [comment]
LGTM

**Comment by anujm1** [comment]
Can you merge please if these look okay?

**Comment by tripzero** [comment]
@anujm1 looks like this needs a rebase.

**Comment by anujm1** [comment]
Done. Can you please pick these up?

**Pull request merged**